src/lhttpc_sock.erl

```diff
@@ -65,9 +65,11 @@
 %%------------------------------------------------------------------------------
 -spec connect(host(), integer(), socket_options(), timeout(), boolean()) ->
     {ok, socket()} | {error, atom()}.
-connect(Host, Port, Options, Timeout, true) ->
+connect(Host0, Port, Options, Timeout, true) ->
+    Host = dns_cache_handler:get_cached_dns(Host0),
```

Comment:
is this a required feature? meaning i'm forced to use it, could it be made optional via the `Options` var?

Reply:
It's not required. Actually, nothing happens if current domain is not in the list of domains to be cached ({domains_to_cache, ["dynamodb.us-west-2.amazonaws.com"]})